src/matrixHelpers/formioLoader/index.js

```diff
@@ -0,0 +1,166 @@
+const initFormioInstance = (formioElem, opts) => {
+  const bodyContainer = $("body");
+  const defaultRedirect = "contact-us/response/";
+  /*
+   * setup config
+   */
+  const baseUrl = `https://${opts.envUrl.trim()}`;
+  const formioContainerId = formioElem.getAttribute("id");
+  const submitBtn = $(`#${formioContainerId} button[name='data[submit]']`);
+  let formName = "";
+  // Check if value is true/exists and is numeric
+  if (opts.form_revision && $.isNumeric(opts.form_revision)) {
```

Comment:
Can you remove isNumeric from this check and remove the console.info message as it not an issue (some form.io forms don't have revisions turned on, we highly suggest they do, and even if it is, then it just uses latest version if not provided)

Also, if the revision is bad, i.e to big or a string, then its up to formio library to complain to the user, not us to 'fix' it for the user and give a false sense of security.